frontend/src/components/Table/PropertyTypeCell.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { CellProps } from 'react-table';
+import { ReactComponent as BuildingSvg } from 'assets/images/icon-business.svg';
+import { ReactComponent as LandSvg } from 'assets/images/icon-lot.svg';
+import { ReactComponent as SubdivisionSvg } from 'assets/images/project-diagram-solid.svg';
+
+import { IProperty } from 'actions/parcelsActions';
+import { PropertyTypes } from 'actions/parcelsActions';
+
+/**
+ * Display an icon based on the property type.
+ * @param {CellProps<IProperty, number>} param0
+ */
+export const PropertyTypeCell = ({ cell: { value } }: CellProps<IProperty, number>) => {
```

Comment:
made this into a reusable component since it is used in tables all over the place.

Reply:
So the component requires a `cell` parameter with a `value` property that is an `IProperty`?

Reply:
Yeah, so essentially it can only be used in tables that are listing rows of IProperty

Reply:
I think that is adequately described in the file name and the above comment, but I could add more doc.